### ISSUE TITLE
Create job/complete route

### ISSUE THREAD

**Issue by jbrown-xentity** [GSA/data.gov#4760]
## User Story

In order to run jobs as often as possible, data.gov harvesters want job completion to notify the flask app/controller so that it can start new jobs waiting in the queue.

## Acceptance Criteria

- [ ] GIVEN the route /job/complete exists \
  WHEN the /job/complete endpoint for the flask app is called (at the end of a job) \
  THEN the controller/flask app kicks off new jobs that are pending.

## Background

In order to not wait for harvester to restart, and to keep jobs moving.

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))

Want to limit this route to authorized only, if possible.

## Sketch

Please note there is a race condition here: when the job makes this post/get request, it is still running. It will wait for a response.

Therefore, we want to give an immediate response, wait 1-2 seconds, and then run the analysis. Otherwise we will find the job still running, and nothing will get kicked off.

**Comment by jbrown-xentity** [comment]
I would not prioritize this highly, this isn't necessary for MVP.

**Comment by neilmb** [comment]
We added `check_for_more_work` so I don't think that this is necessary any more.

**Comment by jbrown-xentity** [comment]
Agreed, I'm closing this. Will not complete, overcome by events (and probably a better pattern).
